tpke/src/lib.rs

```diff
@@ -269,6 +265,7 @@ pub fn setup_simple<E: PairingEngine>(
             g,
             g_inv: E::G1Prepared::from(-g),
             h_inv: E::G2Prepared::from(-h),
+            h, // TODO: Remove this sensitive data here
```

Comment:
What sensitive data?

Reply:
It's a left-over TODO. `h` is just a G1 generator, so it's not sensitive. I didn't know any better when I wrote it, sorry about the confusion.